src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java

```diff
@@ -316,7 +316,8 @@ public JSONObject map(RoboticsWrapper roboticsWrapper) {
             obj.put("otherParties", otherParties);
         }
 
-        if (sscsCaseData.getAppeal().getAppellant().getRole() != null) {
+        if (sscsCaseData.getAppeal().getAppellant().getRole() != null
```

Comment:
Use StringUtils.isNotEmpty to catch empty string

Reply:
Role is not a String though